qiskit_runtime/result/result.py

```diff
@@ -0,0 +1,65 @@
+# This code is part of qiskit-runtime.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Runtime result class"""
+
+import json
+from qiskit.result import Result
+from qiskit.result.postprocess import _hex_to_bin
+from qiskit_runtime.classes import QuasiDistribution
+
+
+class RuntimeResult(Result):
```

Comment:
Same as for the distributions, hopefully there is a common enough usage pattern that one can get away with only a handfull of result objects rather than one per program.  In this case I brought the `RuntimeResult` out top as it has the quasi stuff that might be useful for mitigated results in other programs.  For those programs, they could then use this as their object, or subclass it.

Reply:
I get what you're saying, but I still think calling it `RuntimeResult` is too confusing for the initial release, since all the other programs use completely different output formats.  In the long run I think most runtime programs will be higher level applications that may do error mitigation but will not return quasi probs.